testsupport/nexus-test-common/pom.xml

```diff
@@ -69,8 +69,8 @@
     </dependency>
 
     <dependency>
-      <groupId>org.eclipse.jetty.orbit</groupId>
-      <artifactId>javax.servlet</artifactId>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
```

Comment:
Okay, maybe they gave up on the orbit stuff then?  Seems like each year the specs change gav... fun.